javase/src/main/java/com/google/zxing/client/j2se/ImageReader.java

```diff
@@ -47,7 +50,13 @@ public static BufferedImage readImage(URI uri) throws IOException {
       throw new IOException("Resource not found: " + uri, iae);
     }
     if (result == null) {
-      throw new IOException("Could not load " + uri);
+      Opener opener_ = new Opener();
```

Comment:
Hm, JAI is really old, maybe not. I'm OK with a dependency if it's Maven, licensed reasonably, not too bloated. Also can you describe what kinds of TIFFs ImageIO doesn't read?